libmscore/box.cpp

```diff
@@ -790,11 +790,24 @@ void VBox::layout()
                   contentHeight = minHeight();
 
             setHeight(contentHeight);
+            setBoxHeight(Spatium(contentHeight / spatium()));
```

Comment:
Is this here the cause for the mtest fails?

Reply:
Seems likely, as those now all get that default size set explicitly due to auto-size instead of keeping the old size value.

Reply:
I think I have found a way of doing it without this line. I will try that soon.